src/api-umbrella/web-app/app/models/log_search/elastic_search.rb

```diff
@@ -266,7 +267,7 @@ def aggregate_by_drilldown!(prefix, size = 0)
       :terms => {
         :field => "request_hierarchy",
         :size => size,
-        :include => "^#{Regexp.escape(prefix)}.*",
+        :include => "#{prefix}.*",
```

Comment:
@ThibautGery: Do you recall the reason for this change? According to the [docs](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-bucket-terms-aggregation.html#_filtering_values_2) it appears like the terms aggregations still uses regex values for the `include` argument. But I haven't tested this myself yet, so I can also give it a spin to see what's happening under ElasticSearch 2.